### ISSUE TITLE
Xbox One controllers suddenly no longer connecting 

### ISSUE THREAD

**Issue by droopped** [ndeadly/MissionControl#596]
### Switch Firmware Version

16.0.0 (Latest)

### Atmosphère Version

1.5.0 (Latest)

### Mission Control Version

0.9.0 (Latest)

### Boot Method

Fusée

### Issue Description

Same here, I am on the right firmware (nonBLE) and it was actually working fine a few hours ago, didn't change anything, didn't even power off my console, suddenly mission control doesn't like my Xbox One controllers but still takes a liking to my DS4, I tried 2 model 1708 Xbox One controllers both with nonBLE firmware and neither worked, I'm so confused, I tried rebooting my switch into atmosphere and nothing's changed, help would be greatly appreciated.

**Comment by ndeadly** [comment]
If everything was working as you say, and the controllers are on the old firmware, then my best guess is that the controller configurations have become corrupted. Controllers are disconnected if the virtual controller memory file can't be read for any reason. Try deleting the `/config/MissionControl/controllers` directory and letting Mission Control rebuild everything.

**Comment by mufanvn** [comment]
I have same issue. Did try delete the folder and it worked in handheld mode.
However when I put into the Dock, ít disconnect many times even after I deleted and reconnect.

**Comment by mufanvn** [comment]
Sometime, it even crash the switch and request restart with following message:

Panic occurred while running atmosphere.
Title ID: 010000000000BD00
Error: DABRT < 0x101>

**Comment by ndeadly** [comment]
@mufanvn please provide the error report from the crash. The path to the report is shown beneath the generic error message you posted.

**Comment by mufanvn** [comment]
[report_3034dbd6705.zip](https://github.com/ndeadly/MissionControl/files/11200425/report_3034dbd6705.zip)
Here is the log.

**Comment by mufanvn** [comment]
@ndeadly thank you. I deleted Mission control and reinstalled. It works stable in handheld mode.
However when I put in to dock mode, the controller disconnects quickly. 
I try reconnect in dock mode. It only keep the connection for a short rangre(~1m). When i move away from the Dock, input lag happen and then disconnected.

**Comment by ndeadly** [comment]
@mufanvn this sounds a little like wireless interference to me. Where have you got your dock sitting? Are there any other electronic devices nearby? (besides your tv, obviously). I'm not aware of anything in docked mode that should cause Bluetooth to behave differently.

I'll take a look at the error log later when I'm at a PC

**Comment by ndeadly** [comment]
> @ndeadly thank you for your attention. Near my TV, it has a wifi repeater that may interfere with wireless signals. Let me try using dock mode at another location and get back later.

Absolutely. Wifi also operates on 2.4GHz and would be near the top of my list of possible interference sources

**Comment by ndeadly** [comment]
> After 2-3 times trying, error occurs. I attached the log below
> [report_1536183f9ad.zip](https://github.com/ndeadly/MissionControl/files/11217261/report_1536183f9ad.zip)

This is the same error as before, by the way. Maybe the unexpected disconnects are causing the file to corrupt somehow. I will look into it.

**Comment by mufanvn** [comment]
@ndeadly  I tried another place without wifi repeater near console. Only have my laptop nearby. And on another dock from Done vendor. 
This time it connect successful. However it very lag. Whenever I putsh a button, it takes 1-2 seconds to occur. Sometime it missed input and sometime it is overlap as multiple input even I only push 1 time.
After play for around 15 minutes, it disconnects. I have to delete misson control and reinstall, then connect again. If I don't deleted, it does not detect controller.

P/s: The joy-con still works well in both environment.

**Comment by ndeadly** [comment]
@mufanvn you don't have atmosphere USB3.0 support enabled by any chance do you? It's just been brought to my attention that this can impact wireless connections in docked mode. If you're not sure you can post your `/atmosphere/config/system_settings.ini` file here (if it exists).

**Comment by mufanvn** [comment]
@ndeadly I'm not sure. I attach my system_settings.init
[system_settings.zip](https://github.com/ndeadly/MissionControl/files/11267453/system_settings.zip)

**Comment by mufanvn** [comment]
I tried to change the usb30_force_enabled = u8!0x1 option to usb30_force_enabled = u8!0x0 but it's still not working.
Issue is similar.
